docs/installation/additional_scipts.rst

```diff
@@ -0,0 +1,23 @@
+.. _installation_additional_scripts:
+
+=========================================
+Update Open Forms with additional scripts
+=========================================
+
+Sometimes, upgrading from a specific version of Open Forms to another one requires an extra
+script which is necessary for additional operations. In case this script is missing of your
+container you can follow the instructions to copy it from the project source.
```

Comment:
```suggestion
script which is necessary for additional operations. In case this script is missing in your
container you can follow the instructions to copy it from the project source.
```